src/component/props/SearchProps.tsx

```diff
@@ -0,0 +1,3 @@
+export interface SearchProps {
```

Comment:
props don't have to be in separate files, you can place these types in the same file as the compoenent, it's much more convenient